model/stream.go

```diff
@@ -24,46 +24,67 @@ const (
 type Stream struct {
 	gorm.Model
 
-	Name             string `gorm:"index:,class:FULLTEXT"`
-	Description      string `gorm:"type:text;index:,class:FULLTEXT"`
-	CourseID         uint
-	Start            time.Time `gorm:"not null"`
-	End              time.Time `gorm:"not null"`
-	RoomName         string
-	RoomCode         string
-	EventTypeName    string
-	TUMOnlineEventID uint
-	SeriesIdentifier string `gorm:"default:null"`
-	StreamKey        string `gorm:"not null"`
-	PlaylistUrl      string
-	PlaylistUrlPRES  string
-	PlaylistUrlCAM   string
-	FilePath         string //deprecated
-	LiveNow          bool   `gorm:"not null"`
-	Recording        bool
-	Premiere         bool `gorm:"default:null"`
-	Ended            bool `gorm:"default:null"`
-	Chats            []Chat
-	Stats            []Stat
-	Units            []StreamUnit
-	VodViews         uint `gorm:"default:0"` // todo: remove me before next semester
-	StartOffset      uint `gorm:"default:null"`
-	EndOffset        uint `gorm:"default:null"`
-	LectureHallID    uint `gorm:"default:null"`
-	Silences         []Silence
-	Files            []File `gorm:"foreignKey:StreamID"`
-	Paused           bool   `gorm:"default:false"`
-	StreamName       string
-	Duration         uint32           `gorm:"default:null"`
-	StreamWorkers    []Worker         `gorm:"many2many:stream_workers;"`
-	StreamProgresses []StreamProgress `gorm:"foreignKey:StreamID"`
-	VideoSections    []VideoSection
-	StreamStatus     StreamStatus `gorm:"not null;default:1"`
+	Name              string `gorm:"index:,class:FULLTEXT"`
+	Description       string `gorm:"type:text;index:,class:FULLTEXT"`
+	CourseID          uint
+	Start             time.Time `gorm:"not null"`
+	End               time.Time `gorm:"not null"`
+	RoomName          string
+	RoomCode          string
+	EventTypeName     string
+	TUMOnlineEventID  uint
+	SeriesIdentifier  string `gorm:"default:null"`
+	StreamKey         string `gorm:"not null"`
+	PlaylistUrl       string
+	PlaylistUrlPRES   string
+	PlaylistUrlCAM    string
+	LiveNow           bool `gorm:"not null"`
+	Recording         bool
+	Premiere          bool `gorm:"default:null"`
+	Ended             bool `gorm:"default:null"`
+	Chats             []Chat
+	Stats             []Stat
+	Units             []StreamUnit
+	VodViews          uint `gorm:"default:0"` // todo: remove me before next semester
+	StartOffset       uint `gorm:"default:null"`
+	EndOffset         uint `gorm:"default:null"`
+	LectureHallID     uint `gorm:"default:null"`
+	Silences          []Silence
+	Files             []File `gorm:"foreignKey:StreamID"`
+	ThumbnailInterval uint32 `gorm:"default:null"`
```

Comment:
Would the plugin accept floats? Full seconds might cause some shifting

Reply:
Hm, it is a `number`, see https://github.com/phloxic/videojs-sprite-thumbnails#configuration. I am not sure what happens in the calculation, but it can't hurt to have a float here. :+1: